system/jlib/jstats.cpp

```diff
@@ -1425,7 +1425,7 @@ bool StatsScopeId::setScopeText(const char * text, const char * * _next)
         }
         break;
     case WorkflowScopePrefix[0]:
-        if (MATCHES_CONST_PREFIX(text, WorkflowScopePrefix))
+        if (MATCHES_CONST_PREFIX(text, WorkflowScopePrefix) && isdigit(text[strlen(WorkflowScopePrefix)]))
```

Comment:
minor: should use CONST_STRLEN (more for consistency than performance)

Reply:
I will delete CONST_STRLEN.  C++11 constant folds strlen on constant strings, so the macro is no longer required.